src/app/main-overview/components/OverviewCharts.vue

```diff
@@ -0,0 +1,129 @@
+<script lang="ts" setup>
+import { computed, watch } from 'vue'
+import DoughnutChart from '@/app/common/charts/DoughnutChart.vue'
+import { useStore } from '@/store/store'
+
+const store = useStore()
+
+const isMultizoneMode = computed(() => store.getters['config/getMulticlusterStatus'])
+
+const servicesChartData = computed(() => store.getters.getChart('services', {
+  title: 'Services',
+  showTotal: true,
+}))
+const dataplanesChartData = computed(() => store.getters.getChart('dataplanes', {
+  title: 'DP Proxies',
+  showTotal: true,
+  isStatusChart: true,
+}))
+const meshesChartData = computed(() => store.getters.getChart('meshes', {
+  title: 'Meshes',
+  showTotal: true,
+}))
+const zonesChartData = computed(() => store.getters.getChart('zones', {
+  title: 'Zones',
+  showTotal: true,
+  isStatusChart: true,
+}))
+const zonesCPVersionsChartData = computed(() => store.getters.getChart('zonesCPVersions', {
+  title: 'Zone CP',
+  subtitle: 'versions',
+}))
+const kumaDPVersionsChartData = computed(() => store.getters.getChart('kumaDPVersions', {
+  title: 'Kuma DP',
+  subtitle: 'versions',
+}))
+const envoyVersionsChartData = computed(() => store.getters.getChart('envoyVersions', {
+  title: 'Envoy',
+  subtitle: 'versions',
+}))
+
+watch(() => isMultizoneMode.value, function () {
+  loadData()
+})
+
+loadData()
+
+function loadData() {
+  store.dispatch('fetchMeshInsights')
+  store.dispatch('fetchServices')
+  store.dispatch('fetchZonesInsights', isMultizoneMode.value)
+
+  if (isMultizoneMode.value) {
+    store.dispatch('fetchTotalClusterCount')
+  }
+}
+
+</script>
```

Comment:
**Suggestion**: Please move the script section beneath the template section. I spent a lot of time on consistent code style of our components and all of them are ordered the same way: template, script, style.